pkg/iam/policy.go

```diff
@@ -50,6 +54,20 @@ func (p *PolicyDocument) Add(region, accountID, rolePrefix, username string) {
 	p.Statement = append(p.Statement, statementEntry)
 }
 
+func (p *PolicyDocument) Remove(username string) {
+	awsUserID := usernameToUserId(username)
+
+	newStatements := []StatementEntry{}
+
+	for _, entry := range p.Statement {
+		if entry.Condition.StringLike.AWSUserID != awsUserID {
+			newStatements = append(newStatements, entry)
+		}
+	}
```

Comment:
Are you sure this doesn't end up with the last statement in every case? The classic Go gotcha with reused iteration variable, in this case `entry` is reused and also assigned to the `newStatements` slice. Maybe we can add a test that proves this works as intended?

Reply:
Let's see :) `Statement` contains slices so I would expect to see a bug 🙏

Reply:
I added a test but didn't find a bug. If there was a bug we would end up with a document with the same entry everywhere after a Remove had been called, so the test shows there is no bug, is that understood correctly?

Reply:
Awesome. In that case we are all good 🚀